data/coalition/coalition missions.txt

```diff
@@ -1392,3 +1392,207 @@ mission "Arachi Cartoonist 2"
 				accept
 			label noship
 			`	"Ah, I see. Well, just a reference it would be, so the name matters not, I suppose. Anyway, thanks again Captain."`
+
+
+
+mission "Coalition: Council of Ahr"
+	name "Council of Ahr"
+	description "Bring the representatives of houses Ablomab, Bebliss and Chamgar to <destination> by <date>, so that they may participate in the Council of Ahr."
+	deadline 30
+	to offer
+		has "Coalition: Contributor: done"
+		random < 50
+		"month" < 3
+	source
+		near "Sol Arach" 0 1
+	stopover "Delve of Bloptab"
+	stopover "Hammer of Debrugt"
+	stopover "Stronghold of Flugbu"
+	destination "Ahr"
+	on offer
+		conversation
+			`Shortly after you enter the spaceport, an Arach in a sturdy, red uniform, with golden "pauldrons" on each of his legs' joints, approaches you. "Greetings, Captain <last>," he says. "Interested in taking up some transport jobs, are you? Very well paid, you would be."`
+			choice
+				`	"What are the jobs about?"`
+				`	"Paid how well, exactly?"`
+				`	"Sorry, I'm not looking for that type of job right now."`
+					decline
+			`	"Happening soon, the Council of Ahr is. Gather on the capital, all Arach houses do. Discuss matters related to Arach space, they will, alongside my lieges of the royal family. Paid <payment>, you would be, if to <destination> by <date>, the remaining representatives of the houses, you brought."`
+			choice
+				`	"Sounds simple enough. Who are these representatives?"`
+					goto representatives
+				`	"Can you tell me more about the Council?"`
+				`	"That sounds like it might take a while, and I'm rather busy, so I'll pass."`
+					decline
+			`	"Of course. A few centuries after leaving our home planet, entangled in petty wars, the more prominent houses were. Fearing for the fate of their subjects, called for the first Council, the royal family at the time did. A more civil way to solve disputes, it became, and though war still, many of the houses did, far less frequent, the conflicts were. Now that at peace we are, merely a traditional meeting, it is, taking place every few years, so that discuss how to improve the conditions of Arach space, the houses may."`
```

Comment:
The first choice skips that paragraph, its `goto representatives` makes it so that if you pick it you'll be immediately sent to `label representatives`.

Reply:
Oops, my bad, I completely missed that. Feel free to resolve.